sdk/synapse/Azure.Analytics.Synapse.Spark/tests/samples/Sample1_SubmitSparkJob.cs

```diff
@@ -53,7 +52,13 @@ public void SubmitSparkJobSync()
                 ExecutorCount = 2
             };
 
-            SparkBatchJob jobCreated = client.CreateSparkBatchJob(request);
+            SparkBatchOperation createOperation = client.StartCreateSparkBatchJob(request);
+            while (!createOperation.HasCompleted)
```

Comment:
Just curious: Why not demonstrate the more typical `WaitForCompletion` approach over the manual loop?

Reply:
In another PR I'm going to add async variants for each of these.

Is there a better sync pattern to follow than this?

Reply:
I guess there isn't :) Come to think of it, I'm not sure why Operation<T> doesn't have a `WaitForCompletion`...

Reply:
I wondered that myself....

Reply:
@pakrym may have some insights as to why not.

Reply:
The original idea was that blocking threads synchronously for a long time is a very easy way to starve the thread pool.